static/src/javascripts/projects/common/modules/commercial/membership-messages.js

```diff
@@ -18,115 +22,166 @@ define([
     svgs,
     commercialFeatures,
     userFeatures,
-    defaults,
-    Promise
+    mediator,
+    defaults
 ) {
 
-    var membershipEndpoints = {
-        UK:   'https://membership.theguardian.com/supporter',
-        US:   'https://membership.theguardian.com/us/supporter',
-        AU:   'https://membership.theguardian.com/au/supporter',
-        INT:   'https://membership.theguardian.com/supporter'
-    };
+    return function () {
+        var self = this;
 
-    var defaultData = {
-        arrowWhiteRight: svgs('arrowWhiteRight')
-    };
+        this.id = 'MembershipAndContributionsEngagementBanner';
```

Comment:
I've rebased and moved the entire `membership-messages.js` file into `experiments/tests`